### ISSUE TITLE
Custom CSS fails to load in Point of Sale App & only shows up as content in ui

### ISSUE THREAD

**Issue by lopezdp** [btcpayserver/btcpayserver#4665]
### What is your BTCPay version?

im running `v1.7.12+9344113ae`

### How did you deploy BTCPay Server?

I run on an AWS EC2

### What happened?

im trying to customize my point of sale, and when I add custom css, it shows up in the ui as text instead of modifications to the elements

for example this is what the custom css I entered in the backend looks like when deployed to ui:

<img width="290" alt="Screenshot 2023-02-19 at 1 12 23 AM" src="https://user-images.githubusercontent.com/5707191/219932173-384999c4-afd3-488c-81f3-22e5344dbd3a.png">

### How did you encounter this bug?

I entered custom css in the point of sale settings as such:

<img width="691" alt="Screenshot 2023-02-19 at 1 13 00 AM" src="https://user-images.githubusercontent.com/5707191/219932205-313e15b3-90e7-4fb4-b4d4-2026aac7444a.png">

### Relevant log output

```shell
`2023-02-19 04:21:55.187 +00:00 [WRN] Error while contacting exchange coingecko: The operation was canceled.`
```

### What browser do you use?

Safari Version 16.2 (18614.3.7.1.5)

### Additional information

I have it set to use the default light theme, but on the front end im getting dark theme...?

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by 13Homer13** [comment]
I have similar issue. I can see my custom CSS on the top background of my PoS app. Custom CSS works. Im using firefox 109.0.1.

**Comment by dennisreimann** [comment]
Fixed in #4659, will be released with v1.8 next week.
